src/libpriv/rpmostree-core.c

```diff
@@ -1131,7 +1131,7 @@ rpmostree_context_download_metadata (RpmOstreeContext *self,
                                              G_CALLBACK (on_hifstate_percentage_changed),
                                              NULL);
     g_auto(RpmOstreeProgress) progress = { 0, };
-    rpmostree_output_progress_percent_begin (&progress, "Importing metadata");
+    rpmostree_output_progress_percent_begin (&progress, "Importing rpm-md metadata");
```

Comment:
The "md" there stands for metadata though...a bit like "ATM machine" here (which, I totally understand why people say that, I do it myself sometimes).

How about "Importing rpm repo metadata"?

Reply:
Heh true. I wanted "rpm-md" here since just before this we print e.g. "rpm-md repo 'fedora-updates' (cached); generated: 2018-11-19T21:29:40Z".

How about just "Importing rpm-md" ?

Reply:
Done below! :arrow_down: